src/libraries/sendtohelixhelp.proj

```diff
@@ -163,7 +163,19 @@
     <Message Importance="High" Text="Using WorkItemArchiveWildCard: $(WorkItemArchiveWildCard)" />
     <Message Importance="High" Text="Using Timeout: $(_timeoutSpan)" />
 
-    <ItemGroup>
+    <ItemGroup Condition="'$(TargetOS)' == 'Android'">
+      <_apks Include="$(TestArchiveTestsRoot)**/*.apk" />
```

Comment:
why do we need this extra variable?

Reply:
This is in order to be able to use `%(Filename)` when setting the metadata... it is an MSBuild bug which I can't find at the moment.

Reply:
I will put a comment there about it